sds_data_manager/constructs/ialirt_pointing_schedule_construct.py

```diff
@@ -0,0 +1,95 @@
+"""Cron job to created pointing schedule."""
```

Comment:
This is a super nit, but I think it would be better to refer to _**pointing schedules**_ as a plural thing throughout this file, rather than a singular schedule, because the nature of the job is to create multiple schedules.